### PR TITLE
TC fix: Fixing the SSE test to pass proper appContext for restore validation

### PR DESCRIPTION
**Pull request by sn-px**
**What this PR does / why we need it**:
TC fix: Fixing the SSE test to pass proper appContext for restore validation

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:
After recent pushes to torpedo , the validation function is checking for the proper appContext .before when it was getting masked
Also increased the schedule time just to make sure no further conflict in further backup and restore since i am re-using the namespace 
Logs : https://jenkins.pwx.dev.purestorage.com/job/Users/job/santoshn/job/new-byoc-pipeline/94/consoleFull

Latest log : https://jenkins.pwx.dev.purestorage.com/job/Users/job/santoshn/job/new-byoc-pipeline/97/consoleFull

### CODE REVIEW

File: tests/backup/backup_sse_test.go [hunk 1]
```diff
@@ -316,7 +318,7 @@ var _ = Describe("{CreateBackupAndRestoreForAllCombinationsOfSSES3AndDenyPolicy}
 				log.InfoD("Create a schedule policy")
 				ctx, err := backup.GetAdminCtxFromSecret()
 				log.FailOnError(err, "Fetching px-central-admin ctx")
-				schedulePolicyintervalInMins := 15
+				schedulePolicyintervalInMins := 30
```
Comment (mkoppal-px) [inline tests/backup/backup_sse_test.go:321]: Do not increase the waiting time. We are already taking too long to complete the pipelines.
Keep the interval at 15 min and use different namespaces instead.

Comment (sn-px) [reply tests/backup/backup_sse_test.go:321]: This will not increase run time ..just i am differing the next schedule here ..so that it wont conflict

Comment (sn-px) [reply tests/backup/backup_sse_test.go]: reverted to 15 min

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Review by ak-px — APPROVED** [review]
LGTM

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

**Pull request merged**